src/utils/handlers/authenticationHandlers.js

```diff
@@ -1,7 +1,7 @@
 import axios from 'axios';
 
-import { pathObj, tokenKey } from './../utils/generalVariables';
-import { setHeaders } from './../utils/requestHeaders';
+import { pathObj, tokenKey } from '../../utils/generalVariables';
+import { setHeaders } from '../../utils/requestHeaders';
```

Comment:
Correct me if I'm wrong, but doesn't this path not work with your file restructuring of putting handlers in utils? Right now, you're moving up to `handlers' and then up to `utils`, only to try to access `utils` again?

Reply:
It works but it's weird because it's going up more times than it should just to go back down again. Blame VisualCode and its automatic import updater 🤣 Fixing it

Reply:
Hahaha that's pretty much exactly what I thought as well. Thanks 🔥